### ISSUE TITLE
Standard: is the minimum requirement AA?

### ISSUE THREAD

**Issue by aduggin** [alphagov/accessibility-guidance#2]
WCAG states the following "It is not recommended that Level AAA conformance be required as a general policy for entire sites because it is not possible to satisfy all Level AAA Success Criteria for some content."

However, I think we need to make it clear that many of the AAA success criteria are required as not following them will result in reduced usability and some people being excluded. For example, we definitely require content to be written as clearly and simply as possible and for headings to be used. 

Which of the following can/should we require as part of the minimum standard?
- 1.4.6 Contrast (Enhanced) - The visual presentation of text and images of text has a contrast ratio of at least 7:1
- 1.4.9 Images of Text (No Exception) - Images of text are only used for pure decoration or where a particular presentation of text is essential to the information being conveyed.
- 2.1.3 Keyboard (No Exception) - All functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes.
- 2.2.3 No Timing - Timing is not an essential part of the event or activity presented by the content, except for non-interactive synchronized media and real-time events.
- 2.2.4 Interruptions - Interruptions can be postponed or suppressed by the user, except interruptions involving an emergency
- 2.2.5 Re-authenticating - When an authenticated session expires, the user can continue the activity without loss of data after re-authenticating
- 2.3.2 Three Flashes - Web pages do not contain anything that flashes more than three times in any one second period
- 2.4.9 Link Purpose (Link Only) - A mechanism is available to allow the purpose of each link to be identified from link text alone, except where the purpose of the link would be ambiguous to users in general.
- 2.4.10 Section Headings - Section headings are used to organize the content
- 3.1.3 Unusual Words - A mechanism is available for identifying specific definitions of words or phrases used in an unusual or restricted way, including idioms and jargon
- 3.1.4 Abbreviations - A mechanism for identifying the expanded form or meaning of abbreviations is available
- 3.1.5 Reading Level
- 3.1.6 Pronunciation
- 3.2.5 Change on Request - Changes of context are initiated only by user request or a mechanism is available to turn off such changes
- 3.3.5 Help - Context-sensitive help is available
- 3.3.6 Error Prevention (All)

**Comment by LJWatson** [comment]
We need to think carefully about changing the expected conformance level.

Some Level AAA SC will make things better for one group, whilst making things worse for another (SC1.4.6 for example).

Some Level AAA SC will require significant refactoring of existing services (SC2.2.3 for example).

There is also the general impact of changing a conformance expectation that has been in place across government for nearly a decade (longer if you take WCAG 1.0 into account).

• 1.4.6 Contrast (Enhanced) - The visual presentation of text and images of text has a contrast ratio of at least 7:1

High contrast levels will cause problems for some people. I'm not sure we should make SC1.4.6 a requirement unless we propose to give people the ability to choose a preferred colour scheme.

• 1.4.9 Images of Text (No Exception) - Images of text are only used for pure decoration or where a particular presentation of text is essential to the
information being conveyed.

I think SC1.4.5 is sufficient. We do not use fonts that cannot be created as text given the technologies we use, so the "if the technologies being used can achieve the visual presentation" requirement of 1.4.5 is enough. 

• 2.1.3 Keyboard (No Exception) - All functionality of the content is operable through a keyboard interface without requiring specific timings for individual
keystrokes.

I think SC2.1.1 is sufficient. WCAG provides no additional techniques for meeting SC2.1.3 beyond those provided for SC2.1.1.

• 2.2.3 No Timing - Timing is not an essential part of the event or activity presented by the content, except for non-interactive synchronized media and
real-time events.

I think that SC2.2.1 is sufficient. It either requires that the time limit can be turned off (thus meeting SC2.2.3), or that time limits can be extended by up to 10 times (unless they exceed 20 hours in the first place).

• 2.2.4 Interruptions - Interruptions can be postponed or suppressed by the user, except interruptions involving an emergency

Do we have examples of automatically updating content used on Gov.UK? 

• 2.2.5 Re-authenticating - When an authenticated session expires, the user can continue the activity without loss of data after re-authenticating

• 2.3.2 Three Flashes - Web pages do not contain anything that flashes more than three times in any one second period

Do we have examples of flashing content across Gov.UK?

• 2.4.9 Link Purpose (Link Only) - A mechanism is available to allow the purpose of each link to be identified from link text alone, except where the purpose
of the link would be ambiguous to users in general

Worth including.

• 2.4.10 Section Headings - Section headings are used to organize the content

I think SC1.3.1 is sufficient. The requirement to represent information and relationships covers the need for headings.

• 3.1.3 Unusual Words - A mechanism is available for identifying specific definitions of words or phrases used in an unusual or restricted way, including
idioms and jargon

If we include this, I suggest we would need a central glossary, rather than have individual services provide their own.

• 3.1.4 Abbreviations - A mechanism for identifying the expanded form or meaning of abbreviations is available

• 3.1.5 Reading Level

If content meets the reading age requirement described in <a href="https://www.gov.uk/guidance/content-design/writing-for-gov-uk">writing for Gov.UK</a>, SC3.1.5 doesn't apply.

• 3.1.6 Pronunciation

This SC requires that an explanation is given, where (in context) the pronunciation of a word is the only way of determining its meaning. Do we have examples of where this would apply across Gov.UK?

• 3.2.5 Change on Request - Changes of context are initiated only by user request or a mechanism is available to turn off such changes

Unless we have examples where a change of context is triggered, I think SC3.3.1 and 3.3.2 are sufficient. 

• 3.3.5 Help - Context-sensitive help is available

• 3.3.6 Error Prevention (All)

Unless we have transactions that are not legal, financial, or relevant to user information, I think SC3.3.4 is sufficient.

**Comment by accessiblewebuk** [comment]
I am not aware of any flashing content - but I think it is quite likely there could be some videos containing flashing content. I do think that 2.3.1 at level covers it enough though. Photosensitive epilepsy is uncommon (only a few percent of those who have epilepsy) but of course with government sites small percentages can still be significant numbers. I think that as videos don't auto play, a warning of flashing content would be sufficient. I don't think we can expect government to never publish a video that contains  flash photography for example.

**Comment by accessiblewebuk** [comment]
3.1.3 and 3.1.4 - we probably do cover these within the Digital Service Standard already, but maybe we should consider adding these requirements

**Comment by accessiblewebuk** [comment]
3.3.6 - there are transactions that fall into this category e.g. https://www.gov.uk/check-state-pension - where submitting information returns information but doesn't commit a financial, legal or data change or comittment. However I can't see any benefit from the criterion in that case?

**Comment by LJWatson** [comment]
Would be good to get this list of Level AAA things agreed so I can add them to the relevant bits of the requirements we're building up.

**Comment by aduggin** [comment]
I've booked in a meeting for tomorrow (2 - 3) to for us all to chat and make a decision
